_scicomputing/reference_training.md

```diff
@@ -57,20 +57,70 @@ Several groups on campus host weekly or monthly office hours to provide assistan
+  - [R Programming](https://www.coursera.org/specializations/data-science-foundations-r): A beginner-level program has five mini-courses. It takes about 4 months to complete.
+  - [Statistics with R](https://www.coursera.org/specializations/statistics): A beginner-level program with five mini-courses. It takes about 7 months to complete.
+  - [Genomic Data Science](https://www.coursera.org/specializations/genomic-data-science): An intermediate-level program for those who are already aquainted with R. It has eight mini-courses. It takes about 6 months to complete.
+  - [Python Programming](https://www.coursera.org/specializations/data-science-python): An intermediate-level program that takes about 4 months to complete.
+  - *Coursera offers a 7-day free trial, and is a paid subscription service after*
+
+- [Udacity](https://www.udacity.com): Offers courses on [Data Science](https://www.udacity.com/courses/school-of-data-science)
+  - *Udacity is a paid subscription service*
+  - [Currently offering one month free for their Nanodegree programs.](https://blog.udacity.com/2020/03/one-month-free-on-nanodegrees.html)
+
+- [Udemy](https://www.udemy.com): Offers courses on [Data Science](https://www.udemy.com/courses/development/data-science/)
+  - *Udemy offers courses at various price points.*
+  - Keep an eye out for sales which happen regularly and can drastically reduce the cost.
+
+- [CognitiveClass.ai](https://cognitiveclass.ai/) offers a wide range of courses focused on data science, AI, and cloud computing.
+  - *All courses are free*
+
+- [The Open Source Data Science Masters](http://datasciencemasters.org): An open-source curriculum for learning data science. This is a mixed media course made up of videos, books, and slides.
+  - *Some content is free, some is paid*
+  
 - [CalTech](http://work.caltech.edu/telecourse) Learning from Data
+  - *A free YouTube series*
+
+### Interactive Coding Platforms
+
+- [DataQuest](https://www.dataquest.io/): A subscription service that offers programs and courses focused on data anlysis and engineering in Python and R.
+    -*Teired payment system with basic and premium plans*
+- [CodeAcademy](https://www.codecademy.com/): A subscription service that offers coding programs and courses in many different languages.
+  -*Teired payment system with limited content available for free*
+  
+- [DataCamp](https://www.datacamp.com/): A subscription service that offers coding programs and courses in many different languages.
+  - We previously used DataCamp subscriptions in fredhutch.io courses, but changes in their licenses made this strategy cost prohibitive
+  - Anyone considering DataCamp services should be aware of [this incident involving their CEO](https://www.buzzfeednews.com/article/daveyalba/datacamp-sexual-harassment-metoo-tech-startup) before choosing to pay for their service.
```

Comment:
Happy to be overruled by someone else if this type of thing is important to point out but this doesn't really seem like the platform to be calling out sexual harassment claims (if you dig deeper into the sources you'll see the CEO actually stepped down in [May 2019](https://www.datacamp.com/community/blog/board-update) )... maybe just don't include the DataCamp link if you don't agree with their company?  There are a ton of other interactive coding platforms out there that could be referenced that wouldn't pull controversy onto a training page.

Reply:
I personally support inclusion of this explanation, because I still receive a lot of questions asking about availability of these courses though the Hutch (since we used to use them to teach courses through fredhutch.io). I'm a personal friend of Kara's and know several other former employees and consultants, none of whom work with the group anymore, and I can't in good faith support a company that has a lackluster response to inclusivity (which I hold as a personal core value, and also matches the Hutch's commitment to diversity). 

At the very least, I'd like to include a note saying "we used to have licenses, but no longer"; I'd be interested in others' opinions of how to represent these materials here. Pinging @vortexing for opinions (I know you have some).

Reply:
Hi!
As the contributor of this content, I obviously think it should be included but I figured I could add some context. I want to reiterate what Kate said - that this company has had what could best be described as a lackluster response to sexual harassment by their CEO. I don't believe that providing this information along with other information about the company is controversial. It simply provides important context to those who might choose to give that company their money. 

I've brought this specific incident up with the Office of Diversity and had a good conversation with Ana Prada and Aiko Bethea about it. It might be worthwhile to ping Ana or Paul for their thoughts if we're concerned!

Reply:
I think it would be a great idea to bring this up to the Office of Diversity and get their opinion on what it means for us to include a mention like this in a publicly available website affiliated with the Hutch.

Reply:
I'll ping Ana!

Reply:
ok - we've decided for now to wait on more information before determining how to handle the DataCamp issue. Moving forward with merging the rest of the content.